tools/src/main/java/org/apache/kafka/trogdor/common/ClassUtil.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.trogdor.common;
+
+import org.apache.kafka.common.utils.Utils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+
+/**
+ * Utilities for working with Java classes.
+ */
+public class ClassUtil {
+    private static final Logger log = LoggerFactory.getLogger(ClassUtil.class);
+
+    /**
+     * Construct a new object using a class name and parameters.
+     *
+     * @param className                 The full name of the class to construct.
+     * @param params                    A sequence of (type, object) elements.
+     * @param <T>                       The type of object to construct.
+     * @return                          The new object.
+     * @throws ClassNotFoundException   If there was a problem constructing the object.
+     */
+    public static <T> T construct(String className, Object... params)
+            throws ClassNotFoundException {
+        Class<?>[] argTypes = new Class<?>[params.length / 2];
+        Object[] args = new Object[params.length / 2];
+        try {
+            Class c = Class.forName(className);
```

Comment:
It will be good to specify classloader, just in case this gets used in other projects:
```
Class.forName(className, true, Utils.getContextOrKafkaClassLoader())
```